node/src/services/builder.rs

```diff
@@ -131,8 +132,15 @@ where
     }
 
     /// Get the [`FileStorage`] from the builder.
-    pub fn file_storage(&self) -> &Option<Arc<RwLock<FL>>> {
-        &self.file_storage
+    pub fn rpc_config(&self) -> StorageHubClientRpcConfig<T, FL, FS> {
+        StorageHubClientRpcConfig::new(
+            self.file_storage
+                .clone()
+                .expect("File Storage not initialized"),
+            self.forest_storage
+                .clone()
+                .expect("Forest Storage not initialized"),
+        )
```

Comment:
Thinking about this, maybe it'd be better to keep separate getter methods for `file_storage` and `forest_storage` (they are different components of StorageHub and its Builder. And then we construct the `StorageHubClientRpcConfig` with them.

Reply:
I tend to agree with you. But this way is fine by me either way.